vars/pod.groovy

```diff
@@ -104,6 +106,18 @@ def call(params = [:], Closure body) {
 
     // XXX: look into converting to a YAML string instead
     def label = "pod-${UUID.randomUUID().toString()}"
+    if (params['name']) {
+        def extractedString = ""
+        def regexPattern = /pod-(\w+)-.+/
+        def matcher = (label =~ regexPattern)
+            if (matcher.find()) {
+                extractedString = matcher.group(1)
+                println("extract string: ${extractedString}")
+            }
+        label = "debug-pod-" + params['name'] + "-" + params['arches'] + "-" + extractedString
+        println("LABEL PRINT: ${label}")
```

Comment:
What's the point of this code? If the caller passes in the name then we should just use it?

Reply:
i.e. this whole block should probably be:
```
if (params['name']) {
    label = params['name']
}
```

Reply:
yes, if the caller passes in the name from the debug-pod, then this "if statement' gets invoked. The name is right now passed only from the debug-pod job. The string in the label is then tweaked to print the desired pod name in the output.